### ISSUE TITLE
Warnings about unsupported parameters

### ISSUE THREAD

**Issue by mjankowski** [standardrb/standard-rails#25]
Background: small rails project using standard, with `standard-rails` enabled.

Running standard yields a passing cop check, but some warning output:

```
Warning: Lint/RedundantSafeNavigation does not support AllowedMethods parameter.

Supported parameters are:

  - Enabled
Warning: Style/InvertibleUnlessCondition does not support InverseMethods parameter.

Supported parameters are:

  - Enabled
```

Confirmed that disabling just `standard-rails` removes this warning.

**Comment by askegg** [comment]
I am seeing the same behavior.

```shell
Ruby version: 3.3.0
   Current directory: /Users/andrew/Documents/Project
     RuboCop version: 1.59.0
    Standard version: 1.33.0
Standard config file: /Users/andrew/Documents/Project/.standard.yml
```

My .standard.yml for reference:
```shell
plugins:
  - standard-rails
  - standard-performance
``` 

Gems installed:
```shell
standard (1.33.0)
standard-custom (1.0.2)
standard-performance (1.3.0)
standard-rails (1.0.0)
rubocop (1.59.0)
rubocop-ast (1.30.0)
rubocop-performance (1.20.1)
rubocop-rails (2.23.1)
```

**Comment by jenerts-at-levelpath** [comment]
Seems that solution is on its way - PR and mentioned under this issue: https://github.com/standardrb/standard-rails/issues/23

**Comment by searls** [comment]
I'm a little stumped by how to proceed with this one. 

Here's what I think is happening and why I got myself into trouble with the design of standard-rails and lint_roller.

1. standard-rails works by loading the rubocop-rails yaml and [hand-merging it](https://github.com/standardrb/standard-rails/blob/main/lib/standard/rails/plugin.rb#L36-L45) 
2. `Style/InvertibleUnlessCondition` and `Lint/RedundantSafeNavigation` are both cops shipped with base rubocop and these parameters (`InverseMethods` and `AllowedMethods`) are already populated
3. When using stock rubocop-rails, its [inject_defaults!](https://github.com/rubocop/rubocop-rails/blob/master/lib/rubocop-rails.rb#L14) method will effectively merge its own additional rails-specific values for these options in with the base cops' config (in an additive way)

standard-rails is probably failing here in a couple ways:

1. Because plugins are not allowed to override the config of already-defined cops (as part of the safeguard to ensure standard's config isn't re-configurable), these will probably just be ignored
2. Because the plugin hasn't loaded base rubocop—only the cops shipped with rubocop-rails—the Ruby files specifying these cops is also not loaded yet, which triggers the warning as soon as the config is loaded (it doesn't know about these options because it doesn't know about these cops, is my assumption)

Paths forward:

1. Get way more sophisticated about munging configs for special cases (like a rule genuinely needing more params defined when in "rails" mode, as these do)
2. Breathe a sigh of relief that both of these are actually disabled by standard-rails and just delete them from the config, bypassing this issue and related warnings, but knowing full well this may be biting us elsewhere silently already if similar cops exist and an audit is probably warranted

To get this warning cleared I'm going to follow path 2 for now and cut a release. @camilopayan, what do you think about doing this sort of audit to identify the cops specified in this gems' (or rubocop-rails') config that are defined in base rubocop and ensuring they're (1) all disabled and (2) removed from the config object that the plugin returns to standard

**Comment by searls** [comment]
Landed in 1.0.2
